compiler-plugin/src/main/scala/scalapb/compiler/DescriptorPimps.scala

```diff
@@ -155,7 +155,15 @@ trait DescriptorPimps {
 
     def fieldOptions: FieldOptions = fd.getOptions.getExtension[FieldOptions](Scalapb.field)
 
-    def annotationList = fieldOptions.getAnnotationsList().asScala.toSeq
+    def annotationList: Seq[String] = {
+      val deprecated = {
+        if (fd.getOptions.getDeprecated)
+          List("""@scala.deprecated("", "")""")
```

Comment:
Please change to `deprecated(message="Marked as deprecated in proto file")`  for a friendlier message (presumably it would also make it easier to exclude by regex from various linters). The second parameter can be omitted since it defaults to `""`.

Reply:
done